react/features/transcription/actionTypes.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * The type of (redux) action which indicates that an end point message
+ * sent by another participant to the data channel is received.
+ *
+ * {
+ *     type: ENDPOINT_MESSAGE_RECEIVED,
+ *     conference: JitsiConference,
+ *     participant: Object,
+ *     p: Object
+ * }
+ */
+export const ENDPOINT_MESSAGE_RECEIVED = Symbol('ENDPOINT_MESSAGE_RECEIVED');
+
+/**
+ * The type of (redux) action which indicates that a transcript with
+ * a new message_id is received.
+ *
+ * {
+ *      type: ADD_TRANSCRIPT_MESSAGE,
+ *      transcriptMessageID: string,
+ *      participantName: string
+ * }
+ */
+export const ADD_TRANSCRIPT_MESSAGE = Symbol('ADD_TRANSCRIPT_MESSAGE');
+
+/**
+ * The type of (redux) action which indicates that a transcript with an
+ * existing message_id to be updated is received.
+ * is received.
+ *
+ * {
+ *      type: UPDATE_TRANSCRIPT_MESSAGE,
+ *      transcriptMessageID: string,
+ *      newTranscriptMessage: Object
+ * }
+ */
+export const REMOVE_TRANSCRIPT_MESSAGE = Symbol('REMOVE_TRANSCRIPT_MESSAGE');
```

Comment:
The name does not match the jsdoc

Reply:
Switched UPDATE_TRANSCRIPT_MESSAGE, REMOVE_TRANSCRIPT_MESSAGE.
Updating it.